controllers/scripts/common-env.sh

```diff
@@ -103,8 +103,60 @@ export MAPPED_PORT="$POD_PORT"
 export MAPPED_TLSPORT="$POD_TLSPORT"
 {{- end}}
 
+parseCustomNetworkIP() {
+  local networks=$1
+  networks="$(echo "${networks}" | sed 's/\[//g'| sed 's/\]//g')"
+
+  DATA="$(curl --cacert $CA_CERT -H "Authorization: Bearer $TOKEN" "$KUBE_API_SERVER/api/v1/namespaces/$NAMESPACE/pods/$MY_POD_NAME")"
+
+  NETWORKIPS="$(echo $DATA | python3 -c "import sys, json
+data = json.load(sys.stdin);
+networks = '${networks}';
+networks = networks.strip().split(' ')
+def getNetworkIP(data, networks):
+    annotations = data['metadata']['annotations']
+    key = 'k8s.v1.cni.cncf.io/network-status'
+    if key in annotations:
+      netIPs = []
+      for net in json.loads(annotations[key]):
+        if net['name'] in networks:
+          netIPs.extend(net['ips'])
+      return ' '.join(netIPs)
+print(getNetworkIP(data, networks))")"
```

Comment:
Intentionally kept support for multiple networkNames even though only 1 networkName is allowed in CRD Schema